### PR TITLE
Fixing README code exemples

### PR DESCRIPTION
**Pull request by bernardolm**

### CODE REVIEW

**Comment by gravis** [comment]
I don't understand what are you trying to fix here, you just renamed the import.
That's not necessary

**Comment by bernardolm** [comment]
The exemples does't run or build

File: README.md [hunk 1]
```diff
@@ -58,16 +58,33 @@ type NullFormatter struct {
...
+
+// Don't spend time formatting logs
+func (NullFormatter) Format(e *logrus.Entry) ([]byte, error) {
+    return []byte{}, nil
+}
+
+func main() {
+    var log = logrus.New()
+    log.Infof("Log messages are now sent to Graylog (udp://%s)", "<graylog_ip>:<graylog_port>")      // Give a hint why logs are empty
+    log.Hooks.Add(graylog.NewGraylogHook("<graylog_ip>:<graylog_port>", map[string]interface{}{})) // set graylogAddr accordingly
```
Comment (gravis) [inline README.md]: in this case, you must rename graylogAddr here too

**Comment by gravis** [comment]
Sorry, this PR is really too potato / potato for me, closing.

**Comment by bernardolm** [comment]
hahaha 
Man, your exemples DOES'T WORK!
It's enough of arrogance, bye.

**Pull request closed without merging**